testsuite/std/base64.c

```diff
@@ -0,0 +1,114 @@
+/**
+ * Lovingly ripped off from http://lpmuds.net/smf/index.php?topic=1531.msg8465#msg8465
+ * Minor modifications by Gesslar@ThresholdRPG
+ */
+
+private nosave string b64chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/" ;
+
+string base64encode( string source )
```

Comment:
this implementation will not work with fluffos's utf8 strings,  in order to work correctly,  you need to call 
string_encode(source, "utf8")  to get a buffer, and then base64 encode it as a byte array.